.github/workflows/ci.yml

```diff
@@ -31,10 +31,13 @@ jobs:
     - run: |
         mvn --batch-mode verify sonar:sonar \
         -Dsonar.login=${SONAR_TOKEN} \
-        -Dsonar.host.url=https://sonarcloud.io \
-        -Dsonar.organization=corona-warn-app \
-        -Dsonar.projectKey=corona-warn-app_cwa-verification-server
+        -Dsonar.host.url=${SONAR_URL} \
+        -Dsonar.organization=${SONAR_ORGANIZATION} \
```

Comment:
You could use GH actions env here:

`${{ github.repository }`
`${{ github.repository_owner }}`

https://help.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context

Reply:
This would only require to live with a slightly different project key `corona-warn-app/cwa-verification-server` <> `corona-warn-app_cwa-verification-server`

Reply:
Well `github.repository` is always owner/reponame - so kind of don't fit here...

Reply:
i use bash replace ;)